software/perception/matt_sandbox/matlab/multisense_calib/test_script_vicon.m

```diff
@@ -121,7 +121,7 @@
     end
     channel = event.channel;
     if (strcmp(channel,'BDI_UTIMES'))
```

Comment:
We have a few uses of BDI - POSE_BDI, BDI_UTIMES.
We should replace this with something else. I suggest "HOST" because 'manufacturer' is too long.